src/value.rs

```diff
@@ -60,6 +63,48 @@ impl Value {
       }
     }
   }
+  
+  /// Converts a value into a JSON string for distribution.
+  pub fn to_json(self) -> String {
```

Comment:
No, our `Value` type has nothing to do with serde. Our `Value` type is supposed to be abstract over any data format not just JSON. I also did it like this so we can hyper tune performance. Whether its using `LinearMap` for objects or writing our own `to_json` method.

I don't plan to integrate this type with serde currently because performance around `Value` is critical.